glui_control.cpp

```diff
@@ -313,12 +319,18 @@ int GLUI_Control::char_width(char c)
 { /* Hash table for faster character width lookups - JVK
        Speeds up the textbox a little bit.
   */
-  int hash_index = c % CHAR_WIDTH_HASH_SIZE;
+  /*Killing this with fire.
```

Comment:
Well I'm extremely uncomfortable with superfluous crap. Most controls should be very few bytes at all. Not kilobytes. A checkbox doesn't need a kilobyte "cache" obviously.

That should be obvious. When I resume work on GLUI I'm just going to copy it wholesale and completely rewrite it to be as tiny and free of debris as can be. Then if you want you can try to translate that back into this code base.

I don't have time for the grandstanding here. Or rather my work does not. I'm not sure what the path will be. If they will ever meet again.